### PR TITLE
community/ruby-rmagick: upgrade to 3.0.0

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

**Comment by Ikke** [comment]
@J0WI If this depends on #6870, it should be part of the same PR. If this can be applied separately, then then can remain separate.

**Comment by J0WI** [comment]
It does not depend on #6870.

File: community/ruby-rmagick/APKBUILD [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # Maintainer: Jakub Jirutka <jakub@jirutka.cz>
 pkgname=ruby-rmagick
 _gemname=rmagick
-pkgver=2.16.0
+pkgver=3.0.0
 _pkgver=${pkgver//./-}
 pkgrel=2
```
Comment (andypost) [inline community/ruby-rmagick/APKBUILD]: needs to bump to 0

**Comment by andypost** [comment]
Fixed with c81b4acc

**Pull request closed without merging**